src/components/SearchBlock.tsx

```diff
@@ -6,7 +6,7 @@ import { v4 as uuidv4 } from 'uuid'
 
```

Comment:
Actually, digging a bit more, this definitely looks like a side effect of changes in `5.1.0`. I'm going to go ahead and create an issue to follow up on this.

Reply:
I don't recall needing to manually restart the dev server each time to see changes. I believe that it was the typical refresh the page to see changes.

Reply:
:thinking: -- Super curious if you can reproduce @P-Gill97 . Per the issue I created, it definitely seems like it should be breaking based on recent changes to `vite` :joy: . 

A simple way to test this would be to add a `console.log` to the end of `main.tsx`. Try reloading in Moodle after changing the output string repeatedly. At some point you should hit the problem. If you type `r` -> `enter` in the vite server terminal where you run `npm run dev`, you should see the latest output string.

Reply:
Per engineering check-in today, I will confirm if I hit the same as the above once I have merged this issue.

Reply:
Confirming the behavior above. If I updated my `console.log` output and then refresh the page, the output in the console is not updated unless I type `r` -> `enter` in the vite server terminal where I run `npm run dev`.